pinot-common/src/main/java/org/apache/pinot/common/restlet/resources/ValidDocIdMetadataInfo.java

```diff
@@ -28,14 +28,16 @@ public class ValidDocIdMetadataInfo {
   private final long _totalValidDocs;
   private final long _totalInvalidDocs;
   private final long _totalDocs;
+  private final String _crc;
```

Comment:
(minor) Rename it to `_segmentCRC` to be more specific? Otherwise people might misread it as valid doc id crc

Reply:
fixed